bin/api_diff.ml

```diff
@@ -1,3 +1,5 @@
+open Diffutils
```

Comment:
I'd prefer not to use a global open here. You can eventually use a local one where it makes sense:
```ocaml
let open Diffutils in
...
```